python/dex/interop/jax/apply.py

```diff
@@ -334,10 +334,16 @@ def tuple_arg_string(names, name_to_ty):
   return f"({tuple_ref_string(names)} : {ty})"
 
 def tuple_ref_string(names):
-  return "(" + ", ".join(names) + ")"
+  if len(names) == 1:
+    return names[0]
+  else:
+    return "(" + ", ".join(names) + ")"
```

Comment:
Why is this change necessary?

Reply:
Because without it, the above would emit stuff like `\((thing):(type)). stuff`, trying to imply a one-element tuple.  It seemed easier to avoid the confusion with grouping parens at this level.